_posts/2019-07-29-plonk-stack.md

```diff
@@ -0,0 +1,211 @@
+
+It's my opinion that some people can benefit from using a service-mesh. It is probably the only optional part of the PLONK stack, but I also believe it offers some great benefits at a very low operational cost.
+
+* end-to-end encryption through mutual TLS
+
+Linkerd can encrypt HTTP traffic through the use of mTLS which is enabled by default for all meshed services. OpenFaaS primarily uses HTTP for synchronous function invocations or microservice calls which means that you can get encryption right from your Ingress Controller through to the API Gateway, to the Pod that served your request.
+
+* detailed metrics
+
+One of the reasons people love OpenFaaS is that it adds instrumentation for all functions or microservices. OpenFaaS achieves that by routing all traffic through the API gateway. The API gateway can even be auto-scaled to stop it from becoming a dreaded "Single Point of Failure" (SPOF). Well Linkerd adds even more metrics and comes with great dashboards built-in.
+
+You will love the Linkerd UI and dashboard which you can open up with a single command `linkerd dashboard`.
+
+* traffic shifting
+
+One of the newest features of Linkerd 2.4 is the ability to shift traffic proportionally between two services by assigning them both a weight. This works out of the box with OpenFaaS without any additional need for changes to the project.
+
+Linkerd has a very active and welcoming community. Their primary sponsor is [Buoyant](https://buoyant.io), which was founded by [William Morgan](https://twitter.com/wm) and [Oliver Gould](https://twitter.com/olix0r) after leaving Twitter.
+
+There are many other features a service-mesh can offer, but these are my top features based upon their usefulness to a FaaS or PaaS.
+
+#### OpenFaaS
+
+When I started OpenFaaS in 2016, the project was going up against some very successful projects from huge corporations. Oftentimes the corporations were producing products which were far too complex to use and operate on a daily-basis. Even the newest projects in the Serverless space have been widely criticised for trying to do too much and being spread thin.
```

Comment:
I believe we can cut the last sentence, or merge them into one in some way.

Reply:
If it stays i think it needs to be evidenced in some way - quote a tweet, or link to an article discussing.

Reply:
I'm going to address this comment by re-writing the paragraph.